### DISCUSSION TITLE
Q&A Issue creators being able to assign labels to their own issue

### DISCUSSION THREAD

**Discussion by 9glenda** [eza-community/eza#351] [Q&A, Howto, Help] [answered]
This should be posted in Q&A in discussions

Is there any reason why the issue creator is not able to assign labels to their own issues?

In my opinion assigning labels helps keeping the issues easy to filter and the issue creator often knows what he's doing and is able to choose the right labels.

I don't think this is a important thing to have but I it's quite useful to everyone.

**Comment by cafkafk** [comment, marked as answer]
> Is there any reason why the issue creator is not able to assign labels to their own issues?

I'm not sure we have that kind of granular control, but I'll maybe see if it's possible later.
